app-modules/knowledge-base/src/Models/KnowledgeBaseArticle.php

```diff
@@ -67,6 +67,7 @@ class KnowledgeBaseArticle extends BaseModel implements Auditable, HasMedia
     protected $fillable = [
         'article_details',
         'category_id',
+        'portal_view_count',
```

Comment:
This doesn't need to be fillable as we will always just increment. And it is safer to not be